src/Package/Impl/Repl/Commands/ReplCommandController.cs

```diff
@@ -151,6 +152,7 @@ public ReplCommandController(ITextView textView, ITextBuffer textBuffer)
         }
 
         private void HandleCancel(RCompletionController controller) {
+            Workflow.Operations.CancelAsync().DoNotWait();
```

Comment:
I don't think it is really necessary. We post `interrupt` as well and don't wait.